README.md

```diff
@@ -49,7 +49,7 @@ Thank you for considering and taking the time to contribute! Before contributing
 ### 🚩 Setting up remote :
 
 1. Now you have to set up remote repositories
-2. Type the command `git remote -v` this will list the remote connection to your repo.
+2. Type the command `git remote -v` after getting inside the repo directory, this will list the remote connection to your repo.
```

Comment:
I guess this is understood that one has to cd into repo to access the git cli for the current project.

Adding it will make the readme more verbose, which we can leave for now.

Reply:
Gotcha!